packages/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,232 @@
+import {
+   Flex,
+   Link,
+   Breadcrumb,
+   BreadcrumbItem,
+   Menu,
+   MenuButton,
+   MenuList,
+   MenuItem,
+   MenuItemProps,
+   IconButton,
+   Text,
+   BreadcrumbItemProps,
+   LinkProps,
+} from '@chakra-ui/react';
+import { FaIcon } from '@ifixit/icons';
+import { faChevronRight, faEllipsis } from '@fortawesome/pro-solid-svg-icons';
```

Comment:
@andyg0808 I suspect the main bundle size growth is that these are not deep imports

Reply:
Oooh, interesting point. Trying deep imports in 41cbaeff8d6857be14726947e1a92002b8d076c4.

Reply:
Did it make a difference? If so, I think we might have reached the point at which some anti-pattern tests are worth adding.

Reply:
Not as far as I can tell. Interestingly, not using the new code at all doesn't seem to help either, so I'm not sure what's going on.